content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -7,7 +7,7 @@ weight: 40
 ---
 
 <!-- overview -->
-{{< feature-state for_k8s_version="v1.1" state="beta" >}}
+{{< feature-state for_k8s_version="v1.19" state="stable" >}}
```

Comment:
nit: I think it is not necessary to list a feature state of `stable`.
You could remove this line.

Reply:
When a feature _goes_ stable I think it does help to list the new GA feature state.

The downside is that it eventually requires cleanup (we could automate that toil a bit).